docs/developer/dependencies.rst

```diff
@@ -41,11 +41,27 @@ Run
 Test
 ----
 
-  - all (including optional) dependecies of *Run*
+  - all (including optional) dependencies of *Run*
   - ``mantid-framework`` * (GPL-3.0-or-later) (Mantid interoperability tests are skipped if not present)
```

Comment:
This is wrong. mantid is only utilised by `mantid-framework`. Could you omit please. I know you didn't change this, but while here.

Reply:
This should be removed entirely, since `scippneutron` was extracted?